bin/mkube

```diff
@@ -8,8 +8,7 @@
 # If we're running under WSL then we have to use the Windows native Minikube
 # as the Linux version doesn't work in WSL. Assume WSL is Microsoft's only
 # Linux distro.
-uname -r | grep "Microsoft" > /dev/null
-if  [ $? -ne 0 ]; then
+if uname -r | grep Microsoft > /dev/null; then
```

Comment:
I think this logic needs to be reversed. Note the `if  [ $? -ne 0 ]; then` you have deleted. Now under MacOS for example `MINIKUBE_EXE` gets evalauted to `  MINIKUBE_EXE="${MINIKUBE_EXE:-/mnt/c/Program Files (x86)/Kubernetes/Minikube/minikube.exe}"` at least for me.

Reply:
You are right. Will fix.